system_description/system_description.yaml

```diff
@@ -1,66 +1,125 @@
 accelerator_version:
   description: "Accelerator driver version"
   command: null
-  vendor_command: {}
+  vendor_command:
+      NVIDIA: echo $CUDA_DRIVER_VERSION
```

Comment:
IMO, making this dependent on environment variable isn't a good idea. How do we ensure that all submitters have the same environment variable settings? @xyhuang  @mwawrzos  FYI